packages/backend/server/src/modules/workspaces/resolver.ts

```diff
@@ -137,6 +139,7 @@ export class UpdateWorkspaceInput extends PickType(
 @UseGuards(CloudThrottlerGuard)
 @Auth()
 @Resolver(() => WorkspaceType)
+@UseInterceptors(CacheInterceptor)
```

Comment:
can we make it globally? and skip cache by default, only controllers/resolvers with `Cache` decorator go through that logic